### PR TITLE
Update 2020-10-28-Nagravision-2.md

### PR DESCRIPTION
**Pull request by Aholicknight**
Fixing some grammar issues in the DMCA request.

### CODE REVIEW

**Comment by tigerclaw989** [comment]
Wait helpful pr’s exist for the dmca repository?

File: 2020/10/2020-10-28-Nagravision-2.md [hunk 1]
```diff
@@ -5,35 +5,35 @@ DOMAIN: gist.github.com
...
 
-NAGRA has evidence that the following URLs are infringing Discovery's rights by enabling users to illegally access the Channels, including the copyrighted materials therein.
+NAGRA has evidence that the following URLs are infringing Discovery's rights by enabling users to access the Channels, including the copyrighted materials therein illegally.
 
 https://gist.github.com/juliohenckel/fc92185f6a90dddef4ea38b2e9792526/raw
 
 The sole purpose of these URLs is to grant unauthorized access to one or more of the Channels.
 
-We hereby demand that you immediately remove, disable access, or block access to the above-mentioned unauthorized and infringing URLs, and permanently cease all use of Discovery's intellectual property.
+We, as a result of this demand that you immediately remove, disable access, or block access to those mentioned above unauthorized and infringing URLs, and permanently cease all use of Discovery's intellectual property.
```
Comment (5HT2) [inline 2020/10/2020-10-28-Nagravision-2.md]: `We, as a result of this, `**

Comment (Aholicknight) [reply 2020/10/2020-10-28-Nagravision-2.md]: Oops.

Comment (Aholicknight) [reply 2020/10/2020-10-28-Nagravision-2.md]: I fixed it.

**Comment by iomintz** [comment]
@tigerclaw989 yeah they do but they're all invalid as GitHub doesn't accept outside PRs to this repository.

**Pull request closed without merging**